src/connected/autobinding.ts

```diff
@@ -68,12 +68,7 @@ export class AutoBindingService implements FileSystemSubscriber {
 
   async checkConditionsAndAttemptAutobinding(params: SuggestBindingParams) {
     const bindingSuggestionsPerConfigScope = params.suggestions;
-    const totalSuggestions = [];
-    Object.keys(bindingSuggestionsPerConfigScope).forEach(configScopeId => {
-      totalSuggestions.push(...bindingSuggestionsPerConfigScope[configScopeId]);
-    });
     if (!this.isConnectionConfigured() || // no connections
-     totalSuggestions.length === 0 || // no suggestions
```

Comment:
We want to show the notification even if there are no suggestions (but connection exists)